src/SFA.DAS.Apprenticeships.Web/Controllers/ChangeOfPriceController.cs

```diff
@@ -90,13 +95,100 @@ public async Task<IActionResult> ProviderInitiatedSubmitChange(CreateChangeOfPri
-		private void PopulateProviderInitiatedRouteValues(CreateChangeOfPriceModel model)
+		[HttpGet]
+		[SetNavigationSection(NavigationSection.ManageApprentices)]
+		[Route("provider/{ukprn}/ChangeOfPrice/{apprenticeshipHashedId}/pending")]
+		public async Task<IActionResult> GetViewPendingPriceChangePageProvider(long ukprn, string apprenticeshipHashedId)
+		{
+			var apprenticeshipKey = await _apprenticeshipService.GetApprenticeshipKey(apprenticeshipHashedId);
+			if (apprenticeshipKey == default(Guid))
+			{
+				_logger.LogWarning($"Apprenticeship key not found for apprenticeship with hashed id {apprenticeshipHashedId}");
+				return NotFound();
+			}
+
+			var pendingPriceChange = await _apprenticeshipService.GetPendingPriceChange(apprenticeshipKey);
+			if (pendingPriceChange == null || !pendingPriceChange.HasPendingPriceChange)
+			{
+				_logger.LogWarning($"Pending Apprenticeship Price not found for apprenticeshipKey {apprenticeshipKey}");
+				return NotFound();
+			}
+
+			return View(ProviderViewPendingViewName, new ProviderViewPendingPriceChangeModel(apprenticeshipKey, apprenticeshipHashedId, pendingPriceChange.PendingPriceChange, ukprn));
+		}
+
+        [HttpGet]
+        [Authorize(Policy = nameof(PolicyNames.HasEmployerAccount))]
+        [Route("employer/{employerAccountId}/ChangeOfPrice/{apprenticeshipHashedId}/pending")]
+        public async Task<IActionResult> GetViewPendingPriceChangePageEmployer(string employerAccountId, string apprenticeshipHashedId)
+        {
+            var apprenticeshipKey = await _apprenticeshipService.GetApprenticeshipKey(apprenticeshipHashedId);
+            if (apprenticeshipKey == default(Guid))
+            {
+                _logger.LogWarning($"Apprenticeship key not found for apprenticeship with hashed id {apprenticeshipHashedId}");
+                return NotFound();
+            }
+
+            var pendingPriceChange = await _apprenticeshipService.GetPendingPriceChange(apprenticeshipKey);
+            if (pendingPriceChange == null || !pendingPriceChange.HasPendingPriceChange)
+            {
+                _logger.LogWarning($"Pending Apprenticeship Price not found for apprenticeshipKey {apprenticeshipKey}");
+                return NotFound();
+            }
+
+            var backUrl = _externalEmployerUrlHelper.CommitmentsV2Link("ApprenticeDetails", employerAccountId, apprenticeshipHashedId);
+            return View(EmployerViewPendingViewName, new EmployerViewPendingPriceChangeModel(apprenticeshipKey, apprenticeshipHashedId, pendingPriceChange.PendingPriceChange, employerAccountId, pendingPriceChange.ProviderName, backUrl));
+        }
+
+        [HttpPost]
+        [Authorize(Policy = nameof(PolicyNames.HasEmployerAccount))]
+        [SetNavigationSection(NavigationSection.ManageApprentices)]
+        [Route("provider/{ukprn}/ChangeOfPrice/{apprenticeshipHashedId}/pending")]
```

Comment:
Is it worth breaking this in 2 and having a accept and reject endpoint?

Reply:
Could have 2 different forms, alternatively we could keep the same endpoint but reduce the logic within by creating handles for the different options (this can be done on a later story)